app/models/derived_variables/case_log_variables.rb

```diff
@@ -5,8 +5,8 @@ module DerivedVariables::CaseLogVariables
     "Self-contained flat or bedsit with common facilities" => 2,
     "Shared flat" => 3,
     "Shared house or hostel" => 4,
-    "Bungalow" => 5,
-    "Self-contained house" => 6,
+    "Bungalow" => 6,
```

Comment:
Could we just reference the location model constant here rather than duplicating it so they can't get out of sync again?

Reply:
I mean, we could do, but this actually represents how we should be saving the data in the case log (coming from data dictionary). So when we get a text value, we know how to map it to an integer. If the mapping changes (for whatever reason) in the locations, we still want to use this mapping that we have here. The reason it failed before is because this mapping was just wrong. If this question was in the JSON I'd probably derive the values from there, but we never ask this question, it's always inferred, that's why the mapping is here

Reply:
No. I wouldn't think so, what do you mean by breaks? It would take the string value that we get from the `location.type_of_unit` and map it to the integer that corresponds to it in the case log. It doesn't really matter what numbers it's mapping to in the locations table

Reply:
It wont technically break, but it will inconsistent with Location types. e.g. If in Locations we will remove option "Shared flat" you will still be able to create log with this location type which is incorrect.

Reply:
tweaked it a bit now, it's never a quickfix...